autoload/lsc/complete.vim

```diff
@@ -210,11 +210,11 @@ function! s:FilterAndSort(base, items) abort
   for l:item in a:items
     let l:word = type(l:item) == type({}) ? l:item.word : l:item
     if l:word =~# l:prefix_base
-      call add(l:prefix_case_matches, l:word)
+      call add(l:prefix_case_matches, l:item)
```

Comment:
`l:item` has `user_data` etc keys, so should return `l:item` instead of `l:word` in here, I think.